lms/templates/student_account/finish_auth.html

```diff
@@ -9,32 +9,9 @@
 </%block>
 
 <%block name="headextra">
-
```

Comment:
Is it OK that the "hidden dependency" on `utility.js` is captured via the `js_extra` tag on line 8, above, or should that be done via RequireJS as well?  (`FinishAuthView.js` requires `window.isExternal`, which is provided by utility.js, which is defined as a static:js group in `envs/common.py` just for the use of FinishAuthView)